include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/CaptureResult.kt

```diff
@@ -22,11 +23,11 @@ sealed interface CaptureResult {
       get() = null
 
     override fun toJson(): JsonObject {
-      val json = JsonObject()
-      json.addProperty("type", "recorded")
-      json.addProperty("golden_file_path", goldenFile.absolutePath)
-      json.addProperty("timestamp", timestampNs)
-      return json
+      val recorded = object {
+        val golden_file_path = goldenFile.absolutePath
+        val timestamp = timestampNs
```

Comment:
Writing `Gson().toJson(this)` would result in the following Json string
```
{goldenFile: "...", timestampNs: ....}
```
This does not pass the test because of the key name.
So I created an object with the correct key name.
(Same reason for `Added`, `Changed` and `UnChanged`.)